doc/manual/source/interaction/geoserver_ui.rst

```diff
@@ -91,7 +173,8 @@ It is then possible to clone this repository::
 
     $ geogig clone http://localhost:8080/geoserver/geogig/topp:gisdata-repo gisdata-repo-clone
```

Comment:
these URL's don't seem correct. The `/geogig` endpoint exposes repositories, not datastores, so they should be `/geogig/repos/<repository name>` instead